code/game/objects/items/weapons/manuals.dm

```diff
@@ -601,4 +601,309 @@
                 </ol>
 
 				</body>
-				</html>"}
\ No newline at end of file
+				</html>"}
+
+/obj/item/weapon/book/manual/virology_guide
+	name = "A Crash Course in Virology"
+	icon_state ="bookVirologyGuide"
+	author = "Frederick Chapman Montagnier"
+	title = "A Crash Course in Virology"
+	book_width = 819
+	book_height = 516
+	dat = {"<html>
+				<head>
+				<style>
+				h1 {
+				  font-family: Arial, Helvetica, sans-serif;
+				  text-align:center;
+				  margin-bottom: 0;
+				}
+				h2 {
+				  font-family: Arial, Helvetica, sans-serif;
+				  color: grey;
+				  text-align:center;
+				  margin-top: 0;
+				}
+				h3 {
+				  font-family: Arial, Helvetica, sans-serif;
+				}
+				p,li {
+				  font-family: Arial, Helvetica, sans-serif;
+				  font-size: .95em;
+				}
+				dt {
+				  font-family: Arial, Helvetica, sans-serif;
+				  font-size: .95em;
+				  font-weight: bold;
+				}
+				dd {
+				  font-family: Arial, Helvetica, sans-serif;
+				  font-size: .95em;
+				}
+				table {
+				  font-family: Arial, Helvetica, sans-serif;
+				  border-collapse: collapse;
+				  width:100%;
+				  border: 2px solid black;
+				}
+
+				td, th {
+				  border: 1px solid gray;
+				}
+				</style>
+				</head>
+				<body>
+				<h1>A Crash Course in Virology</h1>
+				<h2>and pathogenics in general</h2>
+				<ul id="menu">
+				  <li><a href="#chapter0">Preamble: HOW DO I CURE VIRUS</a></li>
```

Comment:
These links don't work.

Reply:
Oh right do you have any idea how to fix that? It works in any other browser.

Reply:
I'm afraid not

Reply:
Mind if I keep it that way? it makes the section titles blue and pretty and juicy.

Reply:
Sure, idc